Codec/Compression/Zlib/ByteStringCompat.hs

```diff
@@ -0,0 +1,54 @@
+-- Lifted from the text package, with light editing:
+-- Data.Text.Internal.ByteStringCompat
```

Comment:
@phadej are you all right with this attribution?

Reply:
Yes. That module doesn't really contain anything "copyright-able", just cruft.